packages/yourloops/components/patient-data/patient-data.utils.ts

```diff
@@ -84,6 +86,27 @@ export class PatientDataUtils {
     this.patient = patient
   }
 
+  calculateDashboardDateRange(dateBg: string[]): number {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is this method tested in unit tests? Same question for `getRangeDayInMs()`